controllers/user.go

```diff
@@ -42,6 +44,7 @@ func (c *UserController) New() {
 func (c *UserController) Create() {
 	flash := web.NewFlash()
 	form := forms.RegistrationForm{}
+	redirectpath := ""
```

Comment:
This is inconsistent with `currentPath`. Should this be `redirectPath`?

Reply:
@gutakk fixed on [1e280d2](https://github.com/carryall/go-google-scraper-challenge/pull/31/commits/1e280d2d276185c19440cbd908faa53f3f74337a)